### PR TITLE
Added pause between sending messages. Resolves #40

### LINKED ISSUE #40: Can we add a 'wait' period between each message being sent?
I want to simulate a situation where we have many clients, sending a message once a minute. 

I'd suggest a new parameter that accepts a wait time in seconds, like wait-period and then in the worker, add a thread sleep (or similar) to wait for the required period of time.

Awesome tool by the way, thanks!

### PR DESCRIPTION
**Pull request by Mac351**
This allows stressing an MQTT broker with a lot of open connections, but only sending messages infrequently. It is a good use case to stress a broker with thousands of sensors that send an update every minute, for example, a temperature sensor.

### CODE REVIEW

**Comment by Mac351** [comment]
I've just realised that I didn't follow the standard of having string based time parsing eg: 20s. I'll change it to match that standard.

**Comment by Mac351** [comment]
Updated to treat the pause as a duration

File: main.go [hunk 1]
```diff
@@ -153,7 +154,7 @@ func main() {
 
 	actionTimeout, err := time.ParseDuration(*argTimeout)
 	if err != nil {
-		fmt.Fprintf(os.Stderr, "Failed to parse '--timeout': %q is not a valid duration string. See https://golang.org/pkg/time/#ParseDuration for valid duration strings\n", *argGlobalTimeout)
+		fmt.Fprintf(os.Stderr, "Failed to parse '--timeout': %q is not a valid duration string. See https://golang.org/pkg/time/#ParseDuration for valid duration strings\n", *argTimeout)
```
Comment (hikhvar) [inline main.go:157]: Good catch!

File: main.go [hunk 2]
```diff
@@ -236,6 +237,12 @@ func main() {
 	}
 	testCtx, cancelFunc := context.WithTimeout(context.Background(), globalTimeout)
 
+	pauseBetweenMessages, err := time.ParseDuration(*argPauseBetweenMessages)
+	if err != nil {
+		fmt.Printf("Failed parse '--pause-between-messages': %q is not a valid duration string. See https://golang.org/pkg/time/#ParseDuration for valid duration strings\n", *argPauseBetweenMessages)
```
Comment (hikhvar) [inline main.go:242]: To be consistent, could you please write the error to `os.Stderr` via `fmt.Fprintf`?

Comment (Mac351) [reply main.go]: Yes, sure can. I copied from the example above it, so I'll change that too.

**Comment by hikhvar** [comment]
Thank you for the contribution!

**Comment by Mac351** [comment]
Thanks to you!

**Pull request merged**